app/src/main/java/com/github/se/cyrcle/model/review/ReviewViewModel.kt

```diff
@@ -31,21 +30,39 @@ class ReviewViewModel(
   val userReviews: StateFlow<List<Review?>> = _userReviews
 
   /** Selected parking to review/edit */
-  private val _selectedReviewReports = MutableStateFlow<List<ReviewReport>?>(null)
-  val selectedReviewReports: StateFlow<List<ReviewReport>?> = _selectedReviewReports
+  private val _selectedReviewReports = MutableStateFlow<List<ReviewReport>>(emptyList())
+  val selectedReviewReports: StateFlow<List<ReviewReport>> = _selectedReviewReports
 
   fun addReview(review: Review) {
     reviewRepository.addReview(review, {}, { Log.e("ReviewViewModel", "Error adding review", it) })
   }
 
   fun selectReview(review: Review) {
     _selectedReview.value = review
+    loadReportsForSelectedReview()
   }
 
   fun getNewUid(): String {
     return reviewRepository.getNewUid()
   }
 
+  fun clearSelectedReview() {
```

Comment:
Not used anywhere right now. 
Will it be for the onClick of the to-be-added delete object button that is shown on your PR description's image ?

Reply:
I'm keeping it "in the back pocket" because I'll need it soon in some contexts